scaffolds/Lite/jsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
-    "baseUrl": ".",
+    "baseUrl": "./src",
```

Comment:
We can only point to `./src`as the project's `baseUrl` can only be set to `src` or `node_modules`. Create React App does not support other values at this time. Refer [here](https://github.com/facebook/create-react-app/issues/6850).

Reply:
But when we add page in iceworks, it will generate some code like `import PageA from '@/PageA';`

Reply:
As discussed, `baseAdapter.Router` in `iceworks` will be handling the @ alias prefix in import paths.